components/mm-components/notification/MmNotification.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+    <div role="link" class="mm-notification flex p-4" :class="getBackgroundColor">
+        <div class="mm-notification-display flex mr-4" v-if="checkIfImageOrIcon === 'icon'" :class="getBorderRadius">
+            <i aria-hidden="true" class="m-auto mmi" :class="value.display"></i>
+        </div>
+        <img v-if="checkIfImageOrIcon === 'image'" alt="" :src="value.display" class="mm-notification-display flex mr-4"
+            :class="getBorderRadius" :style="{ 'background-color': 'var(--gray-4)' }" />
+        <div class="flex flex-1 flex-col">
+            <a :href="value.link" class="mm-notification-title label gray-6 regular" v-html="titleValue"
```

Comment:
I believe all links in this component will be internal so the `<a>` needs to change to a `<nuxt-link>`